compose-post.sh

```diff
@@ -9,6 +9,38 @@ ln -sfr /usr/lib/systemd/system/{multi-user,default}.target
 # This is fixed in post-RHEL7 systemd
 ln -sf ../tmp.mount /usr/lib/systemd/system/local-fs.target.wants
 
+# The canonical copy of this is in fedora-coreos-config.  If
+# making changes, please do them there first!
+cat > /usr/libexec/coreos-growpart << 'EOF'
+#!/bin/bash
+set -euo pipefail
+path=$1
+shift
+majmin=$(findmnt -nvr -o MAJ:MIN $path)
+devpath=$(realpath /sys/dev/block/$majmin)
+partition=$(cat $devpath/partition)
+parent_path=$(dirname $devpath)
+parent_device=/dev/$(basename ${parent_path})
+# TODO: make this idempotent, and don't error out if
+# we can't resize.
+growpart ${parent_device} ${partition} || true
+touch /var/lib/coreos-growpart.stamp
+EOF
+chmod a+x /usr/libexec/coreos-growpart
+cat > /usr/lib/systemd/system/coreos-growpart.service <<'EOF'
+[Unit]
+ConditionPathExists=!/var/lib/coreos-growpart.stamp
+[Service]
+ExecStart=/usr/libexec/coreos-growpart /
+RemainAfterExit=yes
+Before=sshd.service
```

Comment:
Isn't `Before=` required to be under the `[Unit]` section?

Reply:
Apparently it's just ignored.

Reply:
Okay, I've submitted coreos/fedora-coreos-config#10.

Reply:
https://github.com/coreos/fedora-coreos-config/pull/11

And updated below :arrow_down: